tests/installation/bootloader.pm

```diff
@@ -26,10 +26,16 @@ sub run() {
     bootmenu_network_source;
     specific_bootmenu_params;
     registration_bootloader_params(utils::VERY_SLOW_TYPING_SPEED);
-    select_bootmenu_language;
-    select_bootmenu_video_mode;
-    # boot
-    send_key "ret";
+    if (!check_var('ARCH', 'ppc64le')) {
```

Comment:
because ppc64le is different
why is apple different from pear ? because it's different fruit same as different architecture